k8s_handle/templating.py

```diff
@@ -18,27 +20,30 @@ class TemplateRenderingError(Exception):
 
 
 def get_template_contexts(file_path):
-    with open(file_path) as f:
-        try:
-            contexts = yaml.safe_load_all(f.read())
-        except Exception as e:
-            raise RuntimeError('Unable to load yaml file: {}, {}'.format(file_path, e))
-
-        for context in contexts:
-            if context is None:
-                continue  # Skip empty YAML documents
-            if 'kind' not in context or context['kind'] is None:
-                raise RuntimeError('Field "kind" not found (or empty) in file "{}"'.format(file_path))
-            if 'metadata' not in context or context['metadata'] is None:
-                raise RuntimeError('Field "metadata" not found (or empty) in file "{}"'.format(file_path))
-            if 'name' not in context['metadata'] or context['metadata']['name'] is None:
-                raise RuntimeError('Field "metadata->name" not found (or empty) in file "{}"'.format(file_path))
-            if 'spec' in context:
-                # INFO: Set replicas = 1 by default for replaces cases in Deployment and StatefulSet
-                if 'replicas' not in context['spec'] or context['spec']['replicas'] is None:
-                    if context['kind'] in ['Deployment', 'StatefulSet']:
-                        context['spec']['replicas'] = 1
-            yield context
+    try:
```

Comment:
Minor bug, can crush. Didn't revealed itself before only due to 1-1 relationships between templating and provisioning.